stream-chat-android-offline/src/test/java/io/getstream/chat/android/offline/integration/BaseConnectedIntegrationTest.kt

```diff
@@ -71,10 +71,10 @@ internal open class BaseConnectedIntegrationTest : BaseDomainTest() {
             backgroundSyncEnabled,
             context,
             offlinePlugin = plugin,
+            retryPolicy = NoRetryPolicy()
```

Comment:
Why this change is needed?

Reply:
Not needed anymore as it will be done by Oleg's PR.